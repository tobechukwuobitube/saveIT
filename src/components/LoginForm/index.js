import React from "react";
import "../../assets/css/form.css";
import { Link } from "react-router-dom";

function LoginForm({ onChange, email, password, onLogin, loading }) {

  return (
    <div id="form-container">
      <form>
        <div className="bg-img">
          <h1>Login Here</h1>
          <label htmlFor="email">Email</label>
          <input
            placeholder="Enter Email"
            id="email"
            name="email"
            type="email"
            value={email}
            onChange={onChange}
            required
          />
          <label htmlFor="password">Password</label>
          <input
            type="password"
            placeholder="Enter Password"
            name="password"
            value={password}
            onChange={onChange}
            required
          />
          {loading ? (
            <button
              className="btn submit-link"
              disabled
              style={{ outline: "none" }}
              onClick={onLogin}
            >
              Login
            </button>
          ) : (
            <button
              className="btn submit-link"
              style={{ outline: "none" }}
              onClick={onLogin}
            >
              Login
            </button>
          )}
          <div className="have-account">
            <p>
              <span>{loading ? <div className="loader" /> : <div />}</span>
              <span>Don&apos;t have account?</span>
              <span>
                <Link className="btn" to="/signup">
                  Sign up
                </Link>
              </span>
            </p>
          </div>
        </div>
      </form>
    </div>
  );
}

export default LoginForm;
