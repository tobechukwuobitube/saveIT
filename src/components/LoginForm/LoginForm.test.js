import React from 'react';
import LoginForm from './index';

test('LoginForm snapshot test', () => {
  const wrapper = shallow(<LoginForm />);
  expect(wrapper).toMatchSnapshot();
});
