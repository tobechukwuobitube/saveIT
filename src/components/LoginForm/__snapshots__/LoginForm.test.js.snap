// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm snapshot test 1`] = `ShallowWrapper {}`;
